packages/examples/sdk-frontend-react/src/app/ChatUITest/ChatViewComponent.tsx

```diff
@@ -20,7 +20,7 @@ const ChatViewComponentTest = () => {
         <ChatView 
         onVerificationFail={() => console.log("BOIIII RETURNNNSSSSS")} 
         chatId='0xf8250D363BD1F25f52F10C21188fe82c68C049c4' 
-
+        Component={() => <div><a href="https://github.com" target="_blank">Test</a></div>}
```

Comment:
any reason the  Component parameter in captalized

Reply:
it was actually giving an error that `Property 'component' does not exist on type 'JSX.IntrinsicElements'` while using small letters and I think since it's a component it will be better to keep it capitalized but let me know what you think about it

Reply:
@KlausMikhaelson  let make the Component to some meaningful name and in small case, also give the type as HTMLDivElement.